.github/workflows/release.yml

```diff
@@ -21,12 +21,15 @@ jobs:
     steps:
       - uses: actions/checkout@v4
         with:
-          token: ${{ secrets.PAT }}
+          token: ${{ secrets.RHACS_BOT_GITHUB_TOKEN }}
       - uses: chainguard-dev/actions/setup-gitsign@main
       - name: Setup git user
+        env:
+          GITHUB_USERNAME: ${{ secrets.RHACS_BOT_GITHUB_USERNAME }}
+          GITHUB_EMAIL: ${{ secrets.RHACS_BOT_GITHUB_EMAIL }}
         run: |
-          git config --global user.email "roxbot@stackrox.com"
-          git config --global user.name "Robot Rox"
+          git config --global user.email "${GITHUB_EMAIL}"
+          git config --global user.name "${GITHUB_USERNAME}"
```

Comment:
Maybe we can remove this step and use the setup gitsign provides?
https://github.com/chainguard-dev/actions/blob/e74da3cd82dab55e35736f1ef031a12fa5b95299/setup-gitsign/action.yml#L108-L111

Reply:
A good suggestion. But I'd like to merge as is and get rid of one more roxbot.